plugins/techdocs-module-addons-contrib/src/setupTests.ts

```diff
@@ -14,3 +14,5 @@
  * limitations under the License.
  */
 import '@testing-library/jest-dom';
+
+Element.prototype.scrollIntoView = jest.fn();
```

Comment:
This mock is needed because `scrollIntoView` is not implemented for `jsdom`.  Jsdom is used by tests. More info on issue: https://github.com/jsdom/jsdom/issues/1695. This seems to be the recommended approach as seen [here](https://github.com/jsdom/jsdom/issues/1695#issuecomment-449931788).